Adactin/src/main/java/com/Adactin/step/definitions/SearchHotelPageSteps.java

```diff
@@ -0,0 +1,111 @@
+        searchHotel.submitHotelDetails();
+    }
+    @Given("I select location from dropdown as user {string}")
+    public void selectLocationJson(String id) {
+        searchHotel.setLocation(entityHelper.getCustomerDataById(id).getLocation());
+    }
+    @Given("I select location {string} from location dropdown in Search hotel page")
+    public void selectLocation(String location) {
+        searchHotel.setLocation(location);
+    }
+
+    @Given("I select hotel details from the dropdown as user {string}")
+    public void selectHotelsJson(String id){
+        searchHotel.setHotels(entityHelper.getCustomerDataById(id).getHotelName());
+           }
+    @Given("I select hotels {string} from the hotels dropdown in Search hotel page")
+    public void selectHotels(String hotels){
+        searchHotel.setHotels(hotels);
+    }
+
+    @Given("I select room type from the dropdown as user {string}")
+    public void selectRoomTypeJson(String id)
+    {
+        searchHotel.setRoomType(entityHelper.getCustomerDataById(id).getRoomtype());
+    }
+    @Given("I select room type {string} from the room type drop down")
+    public void selectRoomType(String roomType)
+    {
+        searchHotel.setRoomType(roomType);
+    }
+    @Given("I select number of rooms from the dropdown for the user {}")
+    public void selectNoOfRoomsJson(String id)
+    {
+        searchHotel.setRoomNum(entityHelper.getCustomerDataById(id).getNoOfRoom());
+    }
+    @Given("I select number of rooms {string} from the drop down")
+    public void selectNoOfRoom(String noOfRoom)
+    {
+        searchHotel.setRoomNum(noOfRoom);
+    }
+    @Given("I select number of adults in the room for the user {string}")
+    public void selectAdultsJson(String id)
+    {
+        searchHotel.setAdultRoom(entityHelper.getCustomerDataById(id).getAdults());
+    }
+    @Given("I select number of adults {string} from the drop down")
+    public void selectAdult(String adult)
+    {
+        searchHotel.setAdultRoom(adult);
+    }
+    @Given("I select the number of children in the room for user {string}")
+    public void selectChildJson(String id)
+    {
+        searchHotel.setChildRoom(entityHelper.getCustomerDataById(id).getChild());
+    }
+    @Given("I select number of children {string} from the drop down")
+    public void selectChild(String child)
+    {
+        searchHotel.setChildRoom(child);
+    }
+    @Given("I select the pick in date for user {string}")
+    public void selectDateInJson(String id)
+    {
+        searchHotel.setDatePickIn(entityHelper.getCustomerDataById(id).getDateIn());
+    }
+    @Given("I select pick in date {string} from the calender")
+    public void selectDateIn(String dateIn)
+    {
+        searchHotel.setDatePickIn(dateIn);
+    }
+    @Given("I select the pick out date for user {string}")
+    public void selectDateOutJson(String id)
+    {
+        searchHotel.setDatePickOut(entityHelper.getCustomerDataById(id).getDateOut());
+    }
```

Comment:
Add line space between two methods and add comments for all methods in step definitions